crates/cxx-qt-lib/src/core/qhash/mod.rs

```diff
@@ -121,9 +124,18 @@ where
         T::remove(self, key)
     }
 
-    /// Returns the value associated with the key.
-    pub fn value(&self, key: &T::Key) -> T::Value {
-        T::value(self, key)
+    /// Returns the value associated with the key if it exists.
+    pub fn value(&self, key: &T::Key) -> Option<T::Value> {
```

Comment:
As noted in [#435](https://github.com/KDAB/cxx-qt/pull/435#discussion_r1103730570) it is probably better to name this `get()` instead of `value()`, as that is more Rust idiomatic.
I guess it depends on if you want to make cxx-qt more Rust idiomatic or more Qt idiomatic (I prefer the former), but in this case the latter wouldn't even be true, as this method has slightly different behaviour compared to the Qt `value()`.

Reply:
Done